doc/source/ray-contribute/writing-code-snippets.rst

```diff
@@ -226,21 +228,21 @@ the `doctest` directive and replace problematic sections with ellipsis. ::
         :options: +ELLIPSIS
 
         >>> import ray
-        >>> ray.data.read_images("s3://anonymous@air-example-data-2/imagenet-sample-images")
+        >>> ray.data.read_images("example://image-datasets/simple")
```

Comment:
Can you explain in the PR description why this change was needed?

Reply:
Done